### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by airon90**
Added info about translation, see #119 
Fixed typo

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,4 +19,18 @@ In order to generate a `google-services.json` configuration file follow these st
 - Download `google-services.json` file.
 - Follow instructions to add file to project.
 - Skip "_Add Firebase SDK step_".
-- Run app to verify that the configuration is picked up correctly.
\ No newline at end of file
+- Run app to verify that the configuration is picked up correctly.
+
+## Translating
+If you want to help the project translating the app, you have to:
```
Comment (tupaschoal) [inline CONTRIBUTING.md]: I'd write "by translating"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,4 +19,18 @@ In order to generate a `google-services.json` configuration file follow these st
...
+
+1. Go in [/app/src/main/res](https://github.com/ccomeaux/boardgamegeek4android/tree/develop/app/src/main/res);
+2. Copy and translate the following files in ```value``` folder:
```
Comment (tupaschoal) [inline CONTRIBUTING.md]: in the ```value```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -18,4 +19,18 @@ In order to generate a `google-services.json` configuration file follow these st
...
+
+- arrays.xml
+- strings.xml
+- strings_notifications.xml
+- strings_prefs.xml
+- strings_sync_service.xml
+
+3. Create a folder named ```value-XX``` where ```XX``` is a valid [ISO 639-1 language code](https://developer.android.com/guide/topics/resources/providing-resources.html#AlternativeResources) and insert translated files there
```
Comment (tupaschoal) [inline CONTRIBUTING.md]: Missing a semicolon or period.

**Comment by airon90** [comment]
@tupaschoal Did I add your proposals? I am not so good in PR on Github

**Comment by tupaschoal** [comment]
The ones I suggested in the code yes, the others you didn't, but since it may need Chris' input, I'll wait.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -18,4 +19,18 @@ In order to generate a `google-services.json` configuration file follow these st
...
 - Follow instructions to add file to project.
 - Skip "_Add Firebase SDK step_".
-- Run app to verify that the configuration is picked up correctly.
\ No newline at end of file
+- Run app to verify that the configuration is picked up correctly.
+
+## Translating
+If you want to help the project by translating the app, you have to:
+
+1. Go in [/app/src/main/res](https://github.com/ccomeaux/boardgamegeek4android/tree/develop/app/src/main/res);
```
Comment (ccomeaux) [inline CONTRIBUTING.md]: Let's make this a relative link, e.g. `app/src/main/res`.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -18,4 +19,18 @@ In order to generate a `google-services.json` configuration file follow these st
...
+2. Copy and translate the following files in the ```value``` folder:
+
+- arrays.xml
+- strings.xml
+- strings_notifications.xml
+- strings_prefs.xml
+- strings_sync_service.xml
+
+3. Create a folder named ```value-XX``` where ```XX``` is a valid [ISO 639-1 language code](https://developer.android.com/guide/topics/resources/providing-resources.html#AlternativeResources) and insert translated files there;
```
Comment (ccomeaux) [inline CONTRIBUTING.md]: `.`, not `;`

**Comment by Cussa** [comment]
I'm always taking a look at the app, but not contributing effectively. However, I would like to bring some ideas about this case.
I believe that would be perfect, especially given the number of users of the app, that you provide an "easier" way to do the translations. Today, only people that know something about code would be able to work with it.
I know that proposes a solution without bringing nothing would be very bad. So, I already came with a possible solution: there is a tool called Weblate that has a hosted site for open source applications. Basically, with that, the person that is doing the translations doesn't have to worry about the code itself, only about the strings.
If you prefer to start a thread to discuss it, I'm more than open to it.

Weblate: https://hosted.weblate.org/

File: CONTRIBUTING.md (hunk 4 above)
Comment (airon90) [reply CONTRIBUTING.md]: [This is a test](app/src/main/res)

**Pull request merged**